This patch makes Donkey Kong 64 work for longer than before.
Now the game doesn't hang after the rap, you can even watch the first
part of the new-game intro!
diff -Naur a/Mupen64Plus-1-5-src/mupen64plus.ini b/Mupen64Plus-1-5-src/mupen64plus.ini
--- a/Mupen64Plus-1-5-src/mupen64plus.ini	2011-05-23 05:51:04.737000099 +0200
+++ b/Mupen64Plus-1-5-src/mupen64plus.ini	2011-05-23 05:53:00.014000098 +0200
@@ -3629,7 +3629,7 @@
 [118E9CE360B97A6F8DAB2C9F30660BF5]
 GoodName=Donkey Kong 64 (E) [!]
 CRC=11936D8C 6F2C4B43
-SaveType=Eeprom 16KB
+SaveType=Eeprom 4KB
 Players=4
 Rumble=Yes
 
@@ -3651,13 +3651,13 @@
 [5B677F4BF93D6578252FCAD2C8CEB637]
 GoodName=Donkey Kong 64 (J) [!]
 CRC=053C89A7 A5064302
-SaveType=Eeprom 16KB
+SaveType=Eeprom 4KB
 Players=4
 
 [98A5836E3A5DA7BD0B5819E7498ACEA2]
 GoodName=Donkey Kong 64 (U) (Kiosk Demo) [!]
 CRC=0DD4ABAB B5A2A91E
-SaveType=Eeprom 16KB
+SaveType=Eeprom 4KB
 
 [843DB032839BF1AA74759F2E39A2ABDC]
 GoodName=Donkey Kong 64 (U) (Kiosk Demo) [b1]
@@ -3677,7 +3677,7 @@
 [9EC41ABF2519FC386CADD0731F6E868C]
 GoodName=Donkey Kong 64 (U) [!]
 CRC=EC58EABF AD7C7169
-SaveType=Eeprom 16KB
+SaveType=Eeprom 4KB
 Players=4
 
 [6B4772AA1743FEAC1919618B3FC6B3E1]
@@ -7708,6 +7708,7 @@
 [AD0F2EC565D7575FB37512BC8DF8A092]
 GoodName=Madden NFL 2002 (U) [!]
 CRC=D7134F8D C11A00B5
+SaveType=Eeprom 16KB
 Players=4
 SaveType=Controller Pack
 
